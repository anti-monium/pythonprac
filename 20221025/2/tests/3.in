print(*list(slide(itt.accumulate([7, 3, 9, 5, 78, 19, 35, 28, 7, 1, 343, 228]), 114)))
