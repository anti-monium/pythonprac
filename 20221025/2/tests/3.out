7 10 19 24 102 121 156 184 191 192 535 763 10 19 24 102 121 156 184 191 192 535 763 19 24 102 121 156 184 191 192 535 763 24 102 121 156 184 191 192 535 763 102 121 156 184 191 192 535 763 121 156 184 191 192 535 763 156 184 191 192 535 763 184 191 192 535 763 191 192 535 763 192 535 763 535 763 763
